src/systemd/sd-lldp.h

```diff
@@ -25,6 +25,7 @@
 #include "sd-event.h"
 
 typedef struct sd_lldp sd_lldp;
+typedef struct tlv_packet sd_lldp_tlv_packet;
```

Comment:
This looks wrong. This should be `typedef struct sd_lldp_tlv_packet sd_lldp_tlv_packet;`